sotodlib/tod_ops/binning.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+import logging
+logger = logging.getLogger(__name__)
+
+def bin_signal(aman, bin_by, signal=None,
+               bin_range=None, bins=100, flags=None):
+    """
+    Bin time-ordered data by the `bin_by` and return the binned signal and its standard deviation.
+
+    Parameters
+    ----------
+    aman : TOD
+        The Axismanager object to be binned.
+    bin_by : array-like
+        The array by which signal is binned. It should has the same samps as aman.
+    signal : str, optional
+        The name of the signal to be binned. Defaults to aman.signal if not specified.
+    bin_range : list or None
+        A list specifying the bin range ([min, max])
+        Default is None, which means bin range is set to [min(bin_by), max(bin_by)] automaticaly by np.histogram.
```

Comment:
These two lines, in sphinx, will render as a single paragraph.  So end the first line with a period.  (And I would prefer that your wrap it all into a single paragraph rather than using the newline to imply a paragraph break.)